uber/models/group.py

```diff
@@ -235,3 +235,23 @@ def requested_hotel_info(self):
                     return attendee.requested_hotel_info
         else:
             return any(a.requested_hotel_info for a in self.attendees)
+
+    @property
+    def physical_address(self):
+        address1 = self.address1.strip()
+        address2 = self.address2.strip()
+        city = self.city.strip()
+        region = self.region.strip()
+        zip_code = self.zip_code.strip()
+        country = self.country.strip()
+
+        country = '' if country == 'United States' else country.strip()
```

Comment:
Huh, this is a pretty unusual thing to do. I understand it's for the comptroller, but I might move this specific change out of the property and into the vendor_comptroller_info function if at all possible (it doesn't look easy, though, so... it's probably fine as-is).